### PR TITLE
Users/aazhou/cleanup

### PR DESCRIPTION
**Pull request by aaronyzhou**

### CODE REVIEW

File: scripts/InputParser.ts [hunk 1]
```diff
@@ -72,23 +67,15 @@ export class InputParser {
...
-
-        if (inputColors.length === 0) {
-            //DefaultColors is a static class wich does the processing of colors.
-            return Colors.getColors(values.length);
+        if(inputColors.length < values.length) {
+            //append default colors if we are given less colors than values
+            return inputColors.concat(Colors.getColors(values.length).slice(inputColors.length));
         } else {
-            return values.map((v, idx) => inputColors[idx] || "");
+            return inputColors.slice(0,values.length);
```
Comment (cschleiden) [inline scripts/InputParser.ts]: spacing

File: scripts/errorView.ts [hunk 2]
```diff
@@ -13,24 +12,31 @@ export class ErrorView {
         var container = $("<div />");
         container.addClass("container");
 
-        // create an icon and text for the error
-        var warning = $("<p />");
-        warning.text(error);
-        warning.attr("title", error);
-        container.append(warning);
+        var table = $("<table></table>");
```
Comment (cschleiden) [inline scripts/errorView.ts]: why a table? don't see any tabular data

**Review by cschleiden — CHANGES_REQUESTED** [review]

File: scripts/errorView.ts (hunk 2 above)
Comment (obvioussean) [reply scripts/errorView.ts]: +1

File: scripts/InputParser.ts (hunk 1 above)
Comment (aaronyzhou) [reply scripts/InputParser.ts]: Fixed

File: scripts/errorView.ts (hunk 2 above)
Comment (aaronyzhou) [reply scripts/errorView.ts]: Fixed

**Review by cschleiden — APPROVED** [review]

**Pull request merged**